google-http-client/src/main/java/com/google/api/client/http/HttpTransport.java

```diff
@@ -129,6 +129,15 @@ public boolean supportsMethod(String method) throws IOException {
     return Arrays.binarySearch(SUPPORTED_METHODS, method) >= 0;
   }
 
+  /**
+   * Returns whether the transport is mTLS.
+   *
+   * @return boolean indicating if the transport is mTLS.
+   */
+  public boolean isMtls() {
+    return false;
+  }
```

Comment:
I don't see anywhere this is being used, or set to anything other than `false`. Do we actually need to expose this?

Reply:
it will be used in the generated client code for api endpoint auto switch (auto switch to mtls endpoint if transport is mtls)